tcl/aws_metadata.tcl

```diff
@@ -0,0 +1,60 @@
+namespace eval qc {
+    namespace export \
+        aws_metadata \
+        aws_metadata_get \
```

Comment:
do `aws_metadata_get` and `aws_metadata_token_refresh` need to be exported?

Only reason to export is if we want caller to be able to call using `aws_metadata_token_refresh` instead of fully qualified `qc::aws_metadata_token_refresh`.

Reply:
Updated